lib/index.js

```diff
@@ -2,13 +2,73 @@
 
 // Load modules
 
-const Http = require('http');
 const Hoek = require('hoek');
 
 
 // Declare internals
 
-const internals = {};
+const internals = {
+    STATUS_CODES: {
+        __proto__: null,
```

Comment:
Probably very slightly because you don't have to look up the prototype chain for misses but that should negligible.That's usually for security, might have seen it as Object.create(null) before.